code/modules/surgery/experimental_dissection.dm

```diff
@@ -24,8 +24,8 @@
 
 /datum/surgery_step/dissection
 	name = "dissection"
-	implements = list(/obj/item/scalpel/augment = 75, /obj/item/scalpel/advanced = 60, /obj/item/scalpel = 45, /obj/item/kitchen/knife = 20, /obj/item/shard = 10)// special tools not only cut down time but also improve probability, doesn't use TOOL_SCALPEL because different scalpels have different probs
-	time = 125
+	implements = list(/obj/item/scalpel/augment = 100, /obj/item/scalpel/advanced = 99, /obj/item/scalpel = 90, /obj/item/kitchen/knife = 45, /obj/item/shard = 25)// special tools not only cut down time but also improve probability, doesn't use TOOL_SCALPEL because different scalpels have different probs
```

Comment:
/obj/item/scalpel/augment refers to the toolset implant, not the alien scalpel. /obj/item/scalpel/alien is what you are looking for.

Reply:
Ahh... so I guess alien is automatic 100?

Ehh. I'll allow augmented scalpel to be a thing... people should be encouraged to get implants and shit anyways.

Reply:
currently, the alien scalpel will be 90 with how you have, because the game will assume that since the alien subtype wasn't defined to just use the normal scalpel's values